src/util/getAllConfigs.js

```diff
@@ -1,21 +1,18 @@
 import defaultConfig from '../../stubs/defaultConfig.stub.js'
 import { flagEnabled } from '../featureFlags'
-import { flatMap, get } from 'lodash'
 
 export default function getAllConfigs(config) {
-  const configs = flatMap([...get(config, 'presets', [defaultConfig])].reverse(), (preset) => {
-    return getAllConfigs(preset)
-  })
+  const configs = (config && config.presets ? config.presets : [defaultConfig])
+    .reverse()
```

Comment:
Should be careful here because `reverse` mutates the array, so we are directly mutating `config.presets` which could have side-effects outside of Tailwind itself.

Reply:
Good catch.